tests/test_maybe.py

```diff
@@ -204,3 +220,17 @@ def to_nothing(_: int) -> Maybe[int]:
 # Lambda versions of the same functions, just for test/type coverage
 sq_lambda: Callable[[int], Maybe[int]] = lambda i: Some(i * i)
 to_nothing_lambda: Callable[[int], Maybe[int]] = lambda _: Nothing()
+
+
+if _RESULT_INSTALLED:
```

Comment:
Let's just always run these tests and remove this conditional. No end user is going to run this, only contributors and GitHub CI, so we'd want these tests to always run.

Reply:
Fair enough. I'll remove this and double-check everything.